source/expressions.tex

```diff
@@ -788,7 +792,9 @@
 q();                                                   // OK: outputs \tcode{1a3.14}
 \end{codeblock}
 \end{example}
-This function call operator or operator template is declared
+
```

Comment:
Yes, but if you read the intermediate sentence, it goes off on a tangent, and returning to the template-parameter later is odd.